### PR TITLE
Avoid calling isLocal with null.

### PR DESCRIPTION
**Pull request by EreMaijala**
Fixes a PHP 8.1 deprecation notice.

### CODE REVIEW

**Review by demiankatz — APPROVED** [review]

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala! I've updated this with a CHANGELOG note. I don't want to merge it yet because it will break the build if we don't merge #19 first. Please take a look at that, and if you're happy with it, please approve and merge it, and then you can merge this subsequently. Once everything is merged and passing, let me know and I will mint release 3.2.0 so we can incorporate this into the main project.

**Comment by EreMaijala** [comment]
@demiankatz Thanks! #19 approved and merged. Latest dev merged here as well, and all seems good, so I'll merge this now. Ready for release then!

**Pull request merged**
